lib/nexmo_developer/app/services/translator/smartling_downloader.rb

```diff
@@ -0,0 +1,55 @@
+module Translator
+  class SmartlingDownloader
+    include ::Translator::Utils
+
+    attr_reader :file_uris
+
+    def self.call(attrs = {})
+      new(attrs).call
+    end
+
+    def initialize(file_uris: [])
+      @file_uris = file_uris || file_paths
+    end
+
+    def call
+      raise ArgumentError, "The 'paths' parameter cannot be empty" unless @file_uris
+
+      @file_uris.map do |uri|
+        locales = get_file_status(file_uri: uri)
+        next unless locales
+
+        locales.map { |locale| download_file(locale: locale, file_uri: uri) }
+      end
+    end
+
+    def file_paths
```

Comment:
can we rename this to `file_uris_from_smartling` and the underlying method to `file_uris`?